lib/matplotlib/backend_bases.py

```diff
@@ -2173,44 +2162,47 @@ def switch_backends(self, FigureCanvasClass):
 
     def mpl_connect(self, s, func):
         """
-        Connect event with string *s* to *func*.  The signature of *func* is::
-
-          def func(event)
-
-        where event is a :class:`matplotlib.backend_bases.Event`.  The
-        following events are recognized
-
-        - 'button_press_event'
-        - 'button_release_event'
-        - 'draw_event'
-        - 'key_press_event'
-        - 'key_release_event'
-        - 'motion_notify_event'
-        - 'pick_event'
-        - 'resize_event'
-        - 'scroll_event'
-        - 'figure_enter_event',
-        - 'figure_leave_event',
-        - 'axes_enter_event',
-        - 'axes_leave_event'
-        - 'close_event'
-
-        For the location events (button and key press/release), if the
-        mouse is over the axes, the variable ``event.inaxes`` will be
-        set to the :class:`~matplotlib.axes.Axes` the event occurs is
-        over, and additionally, the variables ``event.xdata`` and
-        ``event.ydata`` will be defined.  This is the mouse location
-        in data coords.  See
-        :class:`~matplotlib.backend_bases.KeyEvent` and
-        :class:`~matplotlib.backend_bases.MouseEvent` for more info.
-
-        Return value is a connection id that can be used with
-        :meth:`~matplotlib.backend_bases.Event.mpl_disconnect`.
+        Connect event string *s* to *func*.
+
+        Parameters
+        ----------
+        s : str
+            One of the following events strings:
```

Comment:
```suggestion
            One of the following events ids:
```

"Event string" as a bit of a vague term. Maybe also `@cbook._rename_parameter("3.3", "s", "event_id")` (or similar). But that's for another PR. Let's keep this one doc-only.

Reply:
Actually the point is that these are strings and not (e.g.) opaque ids.

Reply:
IMHO "event string" is not a thing. It's not an arbitrary string like in "title string", but one of a set of specific identifiers, referencing an event type. It actually is opaque since it's just a label. An ID does not have to be numeric (To quote wikipedia: "An identifier is a name that identifies (that is, labels the identity of) either a unique object or a unique class of objects")

Reply:
sure, changed (above as well)